src/test/regress/expected/with_modifying.out

```diff
@@ -0,0 +1,127 @@
+-- Tests for modifying CTEs and CTEs in modifications
+SET citus.next_shard_id TO 1502000;
+CREATE SCHEMA with_modifying;
+SET search_path TO with_modifying, public;
+CREATE TABLE with_modifying.t1 (id int, val int);
+SELECT create_distributed_table('t1', 'id');
```

Comment:
I know this is pretty minor, feel free to ignore, but table name `t1` is not a very good name 😄